sql-cli/sql_cli/__main__.py

```diff
@@ -37,111 +41,105 @@ def about() -> None:
-    help="Validate Airflow connection(s) provided in the configuration file for the given environment."
+    help="""
+    Validate Airflow connection(s) provided in the configuration file for the given environment.
+    """
 )
 def validate(
-    environment: str = typer.Argument(
+    project_dir: Path = typer.Argument(
+        None, dir_okay=True, metavar="PATH", help="(Optional) Default: current directory.", show_default=False
+    ),
+    env: str = typer.Option(
         default="default",
-        help="environment to validate",
+        help="(Optional) Environment used to declare the connections to be validated",
     ),
-    connection: str = typer.Argument(
+    connection: str = typer.Option(
         default=None,
-        help="airflow connection id to validate",
+        help="(Optional) Identifier of the connection to be validated. By default checks all the env connections.",
     ),
 ) -> None:
-    validate_connections(environment=environment, connection_id=connection)
+    project_dir_absolute = project_dir.resolve() if project_dir else Path.cwd()
+    project = Project(project_dir_absolute)
+
+    validate_connections(project=project, environment=env, connection_id=connection)
 
 
 @app.command(
     help="""
     Run a workflow locally. This task assumes that there is a local airflow DB (can be a SQLite file), that has been
-    initialized with Airflow tables. Users can also add paths to a connections or variable yaml file which will override
-    existing connections for the test run.
-
-    \b
-    Example of a connections.yaml file:
-
-    \b
-    my_sqlite_conn:
-        conn_id: my_sqlite_conn
-        conn_type: sqlite
-        host: ...
-    my_postgres_conn:
-        conn_id: my_postgres_conn
-        conn_type: postgres
-        ...
+    initialized with Airflow tables.
     """
 )
 def run(
-    directory: Path = typer.Argument(
+    workflow_name: str = typer.Argument(
         default=...,
         show_default=False,
-        exists=True,
-        help="directory containing the sql files.",
+        help="name of the workflow directory within workflows directory.",
     ),
-    dags_directory: Path = typer.Argument(
-        default=...,
-        show_default=False,
-        exists=True,
-        help="directory for the generated DAGs.",
-    ),
-    connection_file: Path = typer.Option(
-        default=None,
-        exists=True,
-        help="path to connections yaml or json file",
+    env: str = typer.Option(
+        metavar="environment",
+        default="default",
+        help="environment to run in",
     ),
-    variable_file: Path = typer.Option(
-        default=None,
-        exists=True,
-        help="path to variables yaml or json file",
+    project_dir: Path = typer.Option(
+        None, dir_okay=True, metavar="PATH", help="(Optional) Default: current directory.", show_default=False
     ),
+    verbose: bool = typer.Option(False, help="Whether to show airflow logs", show_default=True),
```

Comment:
I believe we'll need to make this consistent across our commands - to be addressed as part of #1040